src/main/resources/testchipip/bootrom/bootrom.S

```diff
@@ -37,6 +37,15 @@ wfi_loop: // wait for MSIP interrupt to start program execution
   wfi
   j wfi_loop
 
+// jump to reset vector
+.section .text.hang80, "ax", @progbits
+.globl _hang80
+_hang80:
+  j _hang
+
+.section .text.hangC0, "ax", @progbits
```

Comment:
I think you don't need this other section? If you remove it, the address of the `boot_core` section will be immediately after the `j _hang` instruction... which is fine.

Reply:
You're definitely right... I'll fix that.

Reply:
You should recompile the bootrom images and rebump the CY PR as well.